laravel prepared set

I can see a configuration in `config/set/laravel.php` and a reference to it in `src/ValueObject/Set/SetList.php`, but trying to use the following in my `ecs.php` file:

```
(...)
    ->withPreparedSets(laravel: true)
(...)
```

produces the error:
```
 [ERROR] Unknown named parameter $laravel     
```

I do not see mention of the laravel prepared set in `src/Configuration/ECSConfigBuilder.php` as the rest of the prepared sets do have.

--- a/src/Configuration/ECSConfigBuilder.php
+++ b/src/Configuration/ECSConfigBuilder.php
@@ -181,6 +181,8 @@ final class ECSConfigBuilder
         bool $common = false,
         /** @see SetList::SYMPLIFY */
         bool $symplify = false,
+        /** @see SetList::LARAVEL */
+        bool $laravel = false,
 
         // common sets
         /** @see SetList::ARRAY */
@@ -255,6 +257,10 @@ final class ECSConfigBuilder
 
         if ($symplify) {
             $this->sets[] = SetList::SYMPLIFY;
+        }
+
+        if ($laravel) {
+            $this->sets[] = SetList::LARAVEL;
         }
 
         return $this;

--- a/src/FixerRunner/WhitespacesFixerConfigFactory.php
+++ b/src/FixerRunner/WhitespacesFixerConfigFactory.php
@@ -31,7 +31,12 @@ final class WhitespacesFixerConfigFactory
             $lineEnding = "\n";
         }
 
-        return new WhitespacesFixerConfig($this->resolveIndentation(), $lineEnding);
+        $indentation = $this->resolveIndentation();
+
+        assert($lineEnding !== '');
+        assert($indentation !== '');
+
+        return new WhitespacesFixerConfig($indentation, $lineEnding);
     }
 
     private function resolveIndentation(): string

